### DISCUSSION TITLE
Attenzione: nuovi aggiornamenti

### DISCUSSION THREAD

**Discussion by trinko** [iisgiua/giuaschool#596] [Annunci]
A breve saranno visibili nuovi aggiornamenti che andranno a confluire nella nuova versione 1.5.6

**Si raccomanda però di non installare questi aggiornamenti sino al rilascio della nuova versione, perché potrebbero causare problemi alla gestione ordinaria del registro.**

Il prossimo rilascio della 1.5.6 comprenderà:
- gestione delle classi articolate (gruppi interni alla classe), con firme distinte per la stessa ora di lezione;
- gestione dei gruppi di religione (gruppo religione, gruppo materia alternativa, gruppo che non si avvale), con firme distinte per la stessa ora di lezione;
- gestione moduli per le prove di evacuazione;
- gestione invio per moduli generici da parte di alunni/genitori;
- visualizzazione informazioni alunni trasferiti su pagine del coordinatore e staff;
- visualizzazione dettagli della lettura delle circolari per lo staff;
- visualizzazione composizione del Consiglio di Classe per lo staff;
- aggiornamento delle dipendenze software (Symfony 5.4.*)
- varie correzioni minori.

**Comment by trinko** [comment]
> salve, sa una data orientativa dell'uscita della versione 1.5.6? @trinko

Potrei finire al massimo lunedì, forse prima.
Però dico subito che la 1.5.6 **non va installata** se dovete fare lo scrutinio del trimestre a dicembre, perché i test sulla procedura dello scrutinio li rimando a gennaio.

**Comment by gianlucaaiello56** [comment]
Salve, io ho eseguito l'aggiornamento per errore ed ora la schermata docenti ed alunni non funziona più. Si potrà risolvere con la versione 1.5.6. lunedì ?
